packages/oauth/src/index.ts

```diff
@@ -725,6 +859,20 @@ interface OAuthV2Response extends WebAPICallResult {
   };
 }
 
+export interface OAuthV2ExchangeResponse extends WebAPICallResult {
```

Comment:
I had to add this because of conflicts created in other, pre-existing parts of the code. If someone else wants to take a stab at how to resolve this in another way, again, more than happy to accept a better solution!